src/domain/space/components/spaceDashboardNavigation/dashboardNavigation/DashboardNavigation.tsx

```diff
@@ -203,7 +203,7 @@ const DashboardNavigation = ({
   }
 
   return (
-    <PageContentBlock disablePadding disableGap>
+    <PageContentBlock sx={{ py: gutters(0.5), px: gutters(0.1) }} disablePadding disableGap>
```

Comment:
Again, not this line, but I've been testing kb navigation on space settings and I haven't been able to jump from here:
<img width="1397" height="200" alt="image" src="https://github.com/user-attachments/assets/753e6869-0942-4b26-9152-a3af383d3e95" />
to here:
<img width="1397" height="200" alt="image" src="https://github.com/user-attachments/assets/c3a206cc-cd8b-430e-a29d-17f0e380686f" />